tests/operon-runtime/runtime.test.ts

```diff
@@ -67,7 +67,10 @@ async function dropHelloSystemDB() {
 function configureHelloExample() {
   execSync("npm i");
   execSync("npm run build");
-  execSync("npx knex migrate:up");
+  if (process.env.PGPASSWORD === undefined) {
+    process.env.PGPASSWORD = "dbos";
+  }
```

Comment:
Fix a bug where this suite would fail if the user did not configure `PGPASSWORD` in their environment.